Trello Connector 1.0 Docs (#1250)
mulesoft/docs-connectors: trello/1.0/modules/ROOT/pages/index.adoc

= Trello Connector 1.0 - Mule 4

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

Anypoint Connector for Trello (Trello Connector) enables you to integrate custom applications into Trello to facilitate faster collaboration between teams.

For information about compatibility and fixed issues, see the Trello Connector Release Notes.

== Prerequisites

To use this connector, you must be familiar with:

* The Trello API
* Anypoint Connectors
* Mule runtime engine (Mule)
* Elements and global elements in a Mule flow
* How to create a Mule app using Anypoint Studio

Before creating an app, you must have access to:

* The Trello target resource
* Anypoint Platform

> * Anypoint Studio version 4.2.2 or later

Reviewer: 4.2.2 is the Mule runtime engine version not the version of Anypoint Studio. Could devs confirm if this connector can be used with Studio 7.8?

== Common Use Cases For the Connector

> Use Trello Connector to manage Trello boards, lists, and cards.

Reviewer 2: Customer Case Management
* Send customer case information managed by your support teams to your sales, development and other teams for better and faster collaboration .

Project Collaboration
* Manage key tasks for your project teams across all your IT apps (email, messaging, calendar) in one, streamlined and easy-to-use interface. Make working across teams smoother, especially in a remote work environment.

Reviewer 2: @dhirsch1 added use cases.

== Connection Types

> Trello connections use token-based authentication. Each Trello user is given an API key, which enables the user to generate a token. A user then passes his or her API key and token in the URL for each request.

Reviewer: ```suggestion
Trello connections use token-based authentication. Each Trello user is given an API key, which enables the user to generate a token. Users then passes their API key and token in the URL for each request.
```
